{
  "uid" : "1ed82b51727da355",
  "name" : "Verify text in the 'All Specialists' link",
  "fullName" : "tests.specialists_page_test.TestSpecialistPageText#test_sp_02_05_verify_text_in_link",
  "historyId" : "45dbc9c59b9bd62a24467ed37bde8612",
  "time" : {
    "start" : 1728781322107,
    "stop" : 1728781322107,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x561eccbc602a <unknown>\n#1 0x561ecc8ac5e0 <unknown>\n#2 0x561ecc8a47eb <unknown>\n#3 0x561ecc894a09 <unknown>\n#4 0x561ecc896744 <unknown>\n#5 0x561ecc894cbd <unknown>\n#6 0x561ecc89452f <unknown>\n#7 0x561ecc894422 <unknown>\n#8 0x561ecc892493 <unknown>\n#9 0x561ecc892ada <unknown>\n#10 0x561ecc8aef41 <unknown>\n#11 0x561ecc9408c5 <unknown>\n#12 0x561ecc920b22 <unknown>\n#13 0x561ecc93fd7d <unknown>\n#14 0x561ecc9208c3 <unknown>\n#15 0x561ecc8ee6b3 <unknown>\n#16 0x561ecc8ef68e <unknown>\n#17 0x561eccb90a2b <unknown>\n#18 0x561eccb949b1 <unknown>\n#19 0x561eccb7d225 <unknown>\n#20 0x561eccb95532 <unknown>\n#21 0x561eccb6238f <unknown>\n#22 0x561eccbb4f28 <unknown>\n#23 0x561eccbb50f3 <unknown>\n#24 0x561eccbc4e7c <unknown>\n#25 0x7ff30de94ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"1b4dd773043aee8b7a49f3ccb5be1ac5\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fe0c1151a60>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x561eccbb50f3 \\\\u003Cunknown>\\\\n#24 0x561eccbc4e7c \\\\u003Cunknown>\\\\n#25 0x7ff30de94ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x561eccbc602a <unknown>\nE       #1 0x561ecc8ac5e0 <unknown>\nE       #2 0x561ecc8a47eb <unknown>\nE       #3 0x561ecc894a09 <unknown>\nE       #4 0x561ecc896744 <unknown>\nE       #5 0x561ecc894cbd <unknown>\nE       #6 0x561ecc89452f <unknown>\nE       #7 0x561ecc894422 <unknown>\nE       #8 0x561ecc892493 <unknown>\nE       #9 0x561ecc892ada <unknown>\nE       #10 0x561ecc8aef41 <unknown>\nE       #11 0x561ecc9408c5 <unknown>\nE       #12 0x561ecc920b22 <unknown>\nE       #13 0x561ecc93fd7d <unknown>\nE       #14 0x561ecc9208c3 <unknown>\nE       #15 0x561ecc8ee6b3 <unknown>\nE       #16 0x561ecc8ef68e <unknown>\nE       #17 0x561eccb90a2b <unknown>\nE       #18 0x561eccb949b1 <unknown>\nE       #19 0x561eccb7d225 <unknown>\nE       #20 0x561eccb95532 <unknown>\nE       #21 0x561eccb6238f <unknown>\nE       #22 0x561eccbb4f28 <unknown>\nE       #23 0x561eccbb50f3 <unknown>\nE       #24 0x561eccbc4e7c <unknown>\nE       #25 0x7ff30de94ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728781182641,
      "stop" : 1728781182641,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1728781322107,
      "stop" : 1728781322897,
      "duration" : 790
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728781322897,
      "stop" : 1728781323491,
      "duration" : 594
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x561eccbc602a <unknown>\n#1 0x561ecc8ac5e0 <unknown>\n#2 0x561ecc8a47eb <unknown>\n#3 0x561ecc894a09 <unknown>\n#4 0x561ecc896744 <unknown>\n#5 0x561ecc894cbd <unknown>\n#6 0x561ecc89452f <unknown>\n#7 0x561ecc894422 <unknown>\n#8 0x561ecc892493 <unknown>\n#9 0x561ecc892ada <unknown>\n#10 0x561ecc8aef41 <unknown>\n#11 0x561ecc9408c5 <unknown>\n#12 0x561ecc920b22 <unknown>\n#13 0x561ecc93fd7d <unknown>\n#14 0x561ecc9208c3 <unknown>\n#15 0x561ecc8ee6b3 <unknown>\n#16 0x561ecc8ef68e <unknown>\n#17 0x561eccb90a2b <unknown>\n#18 0x561eccb949b1 <unknown>\n#19 0x561eccb7d225 <unknown>\n#20 0x561eccb95532 <unknown>\n#21 0x561eccb6238f <unknown>\n#22 0x561eccbb4f28 <unknown>\n#23 0x561eccbb50f3 <unknown>\n#24 0x561eccbc4e7c <unknown>\n#25 0x7ff30de94ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728781322897,
        "stop" : 1728781323490,
        "duration" : 593
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x561eccbc602a <unknown>\n#1 0x561ecc8ac5e0 <unknown>\n#2 0x561ecc8a47eb <unknown>\n#3 0x561ecc894a09 <unknown>\n#4 0x561ecc896744 <unknown>\n#5 0x561ecc894cbd <unknown>\n#6 0x561ecc89452f <unknown>\n#7 0x561ecc894422 <unknown>\n#8 0x561ecc892493 <unknown>\n#9 0x561ecc892ada <unknown>\n#10 0x561ecc8aef41 <unknown>\n#11 0x561ecc9408c5 <unknown>\n#12 0x561ecc920b22 <unknown>\n#13 0x561ecc93fd7d <unknown>\n#14 0x561ecc9208c3 <unknown>\n#15 0x561ecc8ee6b3 <unknown>\n#16 0x561ecc8ef68e <unknown>\n#17 0x561eccb90a2b <unknown>\n#18 0x561eccb949b1 <unknown>\n#19 0x561eccb7d225 <unknown>\n#20 0x561eccb95532 <unknown>\n#21 0x561eccb6238f <unknown>\n#22 0x561eccbb4f28 <unknown>\n#23 0x561eccbb50f3 <unknown>\n#24 0x561eccbc4e7c <unknown>\n#25 0x7ff30de94ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"1b4dd773043aee8b7a49f3ccb5be1ac5\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728781323525,
      "stop" : 1728781323587,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Specialists Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "specialists_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestSpecialistsPage"
  }, {
    "name" : "host",
    "value" : "fv-az585-604"
  }, {
    "name" : "thread",
    "value" : "2289-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.specialists_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 12,
        "skipped" : 7,
        "passed" : 330,
        "unknown" : 0,
        "total" : 349
      },
      "items" : [ {
        "uid" : "fa843d39974bf5a0",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/150//#testresult/fa843d39974bf5a0",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56303362702a <unknown>\n#1 0x56303330d5e0 <unknown>\n#2 0x5630333057eb <unknown>\n#3 0x5630332f5a09 <unknown>\n#4 0x5630332f7744 <unknown>\n#5 0x5630332f5cbd <unknown>\n#6 0x5630332f552f <unknown>\n#7 0x5630332f5422 <unknown>\n#8 0x5630332f3493 <unknown>\n#9 0x5630332f3ada <unknown>\n#10 0x56303330ff41 <unknown>\n#11 0x5630333a18c5 <unknown>\n#12 0x563033381b22 <unknown>\n#13 0x5630333a0d7d <unknown>\n#14 0x5630333818c3 <unknown>\n#15 0x56303334f6b3 <unknown>\n#16 0x56303335068e <unknown>\n#17 0x5630335f1a2b <unknown>\n#18 0x5630335f59b1 <unknown>\n#19 0x5630335de225 <unknown>\n#20 0x5630335f6532 <unknown>\n#21 0x5630335c338f <unknown>\n#22 0x563033615f28 <unknown>\n#23 0x5630336160f3 <unknown>\n#24 0x563033625e7c <unknown>\n#25 0x7f4875894ac3 <unknown>",
        "time" : {
          "start" : 1728694647678,
          "stop" : 1728694647678,
          "duration" : 0
        }
      }, {
        "uid" : "8a37fbec04da12c5",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/149//#testresult/8a37fbec04da12c5",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55ab1d0c002a <unknown>\n#1 0x55ab1cda65e0 <unknown>\n#2 0x55ab1cd9e7eb <unknown>\n#3 0x55ab1cd8ea09 <unknown>\n#4 0x55ab1cd90744 <unknown>\n#5 0x55ab1cd8ecbd <unknown>\n#6 0x55ab1cd8e52f <unknown>\n#7 0x55ab1cd8e422 <unknown>\n#8 0x55ab1cd8c493 <unknown>\n#9 0x55ab1cd8cada <unknown>\n#10 0x55ab1cda8f41 <unknown>\n#11 0x55ab1ce3a8c5 <unknown>\n#12 0x55ab1ce1ab22 <unknown>\n#13 0x55ab1ce39d7d <unknown>\n#14 0x55ab1ce1a8c3 <unknown>\n#15 0x55ab1cde86b3 <unknown>\n#16 0x55ab1cde968e <unknown>\n#17 0x55ab1d08aa2b <unknown>\n#18 0x55ab1d08e9b1 <unknown>\n#19 0x55ab1d077225 <unknown>\n#20 0x55ab1d08f532 <unknown>\n#21 0x55ab1d05c38f <unknown>\n#22 0x55ab1d0aef28 <unknown>\n#23 0x55ab1d0af0f3 <unknown>\n#24 0x55ab1d0bee7c <unknown>\n#25 0x7f1472894ac3 <unknown>",
        "time" : {
          "start" : 1728608252542,
          "stop" : 1728608252542,
          "duration" : 0
        }
      }, {
        "uid" : "5c2541db9ddb702c",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/322//#testresult/5c2541db9ddb702c",
        "status" : "passed",
        "time" : {
          "start" : 1728557358566,
          "stop" : 1728557358607,
          "duration" : 41
        }
      }, {
        "uid" : "86b48e265c6f6c64",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/321//#testresult/86b48e265c6f6c64",
        "status" : "passed",
        "time" : {
          "start" : 1728554229885,
          "stop" : 1728554229925,
          "duration" : 40
        }
      }, {
        "uid" : "c7fccef42ebdbda3",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/148//#testresult/c7fccef42ebdbda3",
        "status" : "passed",
        "time" : {
          "start" : 1728522232086,
          "stop" : 1728522232138,
          "duration" : 52
        }
      }, {
        "uid" : "495745e6527addea",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/320//#testresult/495745e6527addea",
        "status" : "passed",
        "time" : {
          "start" : 1728470632158,
          "stop" : 1728470632202,
          "duration" : 44
        }
      }, {
        "uid" : "cdf75e1e9ba4fab1",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/319//#testresult/cdf75e1e9ba4fab1",
        "status" : "passed",
        "time" : {
          "start" : 1728468058197,
          "stop" : 1728468058242,
          "duration" : 45
        }
      }, {
        "uid" : "8fb6f036a134814c",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/147//#testresult/8fb6f036a134814c",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728435895179,
          "stop" : 1728435895179,
          "duration" : 0
        }
      }, {
        "uid" : "cc3f77e4078a79cc",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/318//#testresult/cc3f77e4078a79cc",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728378104985,
          "stop" : 1728378104985,
          "duration" : 0
        }
      }, {
        "uid" : "c85e4c255c181ba0",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/317//#testresult/c85e4c255c181ba0",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728375313662,
          "stop" : 1728375313662,
          "duration" : 0
        }
      }, {
        "uid" : "b422e38be57b0fe7",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/316//#testresult/b422e38be57b0fe7",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728375195225,
          "stop" : 1728375195225,
          "duration" : 0
        }
      }, {
        "uid" : "5dd7da56b5f89c23",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/146//#testresult/5dd7da56b5f89c23",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728349283956,
          "stop" : 1728349283956,
          "duration" : 0
        }
      }, {
        "uid" : "33cdc2afeff4eb26",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/315//#testresult/33cdc2afeff4eb26",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728293033996,
          "stop" : 1728293033996,
          "duration" : 0
        }
      }, {
        "uid" : "b9ce9a758105e29f",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/314//#testresult/b9ce9a758105e29f",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728290083567,
          "stop" : 1728290083567,
          "duration" : 0
        }
      }, {
        "uid" : "dd5e9d2b63de5c9b",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/145//#testresult/dd5e9d2b63de5c9b",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator ('xpath', \"//div[contains(@class, 'mt-20')]/a\")\nStacktrace:\n#0 0x55c3c93e813a <unknown>\n#1 0x55c3c90ce5e0 <unknown>\n#2 0x55c3c911dbe8 <unknown>\n#3 0x55c3c911de81 <unknown>\n#4 0x55c3c91648c4 <unknown>\n#5 0x55c3c9142b4d <unknown>\n#6 0x55c3c9161d7d <unknown>\n#7 0x55c3c91428c3 <unknown>\n#8 0x55c3c91106b3 <unknown>\n#9 0x55c3c911168e <unknown>\n#10 0x55c3c93b2b3b <unknown>\n#11 0x55c3c93b6ac1 <unknown>\n#12 0x55c3c939f335 <unknown>\n#13 0x55c3c93b7642 <unknown>\n#14 0x55c3c938449f <unknown>\n#15 0x55c3c93d7038 <unknown>\n#16 0x55c3c93d7203 <unknown>\n#17 0x55c3c93e6f8c <unknown>\n#18 0x7eff99e94ac3 <unknown>",
        "time" : {
          "start" : 1728263565771,
          "stop" : 1728263586281,
          "duration" : 20510
        }
      }, {
        "uid" : "592f55112dcbc323",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/313//#testresult/592f55112dcbc323",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator ('xpath', \"//div[contains(@class, 'mt-20')]/a\")\nStacktrace:\n#0 0x55ec0703c13a <unknown>\n#1 0x55ec06d225e0 <unknown>\n#2 0x55ec06d71be8 <unknown>\n#3 0x55ec06d71e81 <unknown>\n#4 0x55ec06db88c4 <unknown>\n#5 0x55ec06d96b4d <unknown>\n#6 0x55ec06db5d7d <unknown>\n#7 0x55ec06d968c3 <unknown>\n#8 0x55ec06d646b3 <unknown>\n#9 0x55ec06d6568e <unknown>\n#10 0x55ec07006b3b <unknown>\n#11 0x55ec0700aac1 <unknown>\n#12 0x55ec06ff3335 <unknown>\n#13 0x55ec0700b642 <unknown>\n#14 0x55ec06fd849f <unknown>\n#15 0x55ec0702b038 <unknown>\n#16 0x55ec0702b203 <unknown>\n#17 0x55ec0703af8c <unknown>\n#18 0x7f7c53094ac3 <unknown>",
        "time" : {
          "start" : 1728226732793,
          "stop" : 1728226753293,
          "duration" : 20500
        }
      }, {
        "uid" : "ae2e0a0a33cd3cf4",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/312//#testresult/ae2e0a0a33cd3cf4",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator ('xpath', \"//div[contains(@class, 'mt-20')]/a\")\nStacktrace:\n#0 0x56267e7b513a <unknown>\n#1 0x56267e49b5e0 <unknown>\n#2 0x56267e4eabe8 <unknown>\n#3 0x56267e4eae81 <unknown>\n#4 0x56267e5318c4 <unknown>\n#5 0x56267e50fb4d <unknown>\n#6 0x56267e52ed7d <unknown>\n#7 0x56267e50f8c3 <unknown>\n#8 0x56267e4dd6b3 <unknown>\n#9 0x56267e4de68e <unknown>\n#10 0x56267e77fb3b <unknown>\n#11 0x56267e783ac1 <unknown>\n#12 0x56267e76c335 <unknown>\n#13 0x56267e784642 <unknown>\n#14 0x56267e75149f <unknown>\n#15 0x56267e7a4038 <unknown>\n#16 0x56267e7a4203 <unknown>\n#17 0x56267e7b3f8c <unknown>\n#18 0x7f717e094ac3 <unknown>",
        "time" : {
          "start" : 1728221832402,
          "stop" : 1728221852418,
          "duration" : 20016
        }
      }, {
        "uid" : "acda97cd7fca129e",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/144//#testresult/acda97cd7fca129e",
        "status" : "passed",
        "time" : {
          "start" : 1728177063514,
          "stop" : 1728177063561,
          "duration" : 47
        }
      }, {
        "uid" : "3d28b8161cfee08d",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/311//#testresult/3d28b8161cfee08d",
        "status" : "passed",
        "time" : {
          "start" : 1728117696831,
          "stop" : 1728117696882,
          "duration" : 51
        }
      }, {
        "uid" : "a15f6781aa95c9cb",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/310//#testresult/a15f6781aa95c9cb",
        "status" : "passed",
        "time" : {
          "start" : 1728109075847,
          "stop" : 1728109075891,
          "duration" : 44
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "1ed82b51727da355.json",
  "parameterValues" : [ ]
}